### ISSUE TITLE
can this addon support send to firefox?

### ISSUE THREAD

**Issue by EruditePig** [danny0838/webarchiveviewer#3]
When I explore a maff file in my explore, I want to use android firefox to open it. The only success way is click android firefox menu, select webarchiveviewer, find the maff  file, then the file will display.

thank you for your brilliant addon!

**Comment by danny0838** [comment]
No, it's currently impossible due to the restriction of the WebExtension framework.

Additionally, this addon is part of [Web ScrapBook](https://github.com/danny0838/webscrapbook) and is no more maintained. Use Web ScrapBook instead.
